### DISCUSSION TITLE
Many errors "Repository is not accessible. Missing credentials to access repository"

### DISCUSSION THREAD

**Discussion by monisnap-julien** [codemagic-ci-cd/codemagic-docs#2395] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1. Start new build

### Expected results

The build starts

### Actual results

We have an error "Repository is not accessible. Missing credentials to access repository" of "invalid_token"

### Build id (optional)

65df44f16f0e912215289c19

**Comment by dtrdic** [comment, marked as answer]
Hello @monisnap-julien, sounds like your git integration have been disconnected, can you try checking on Teams > team name > team integrations > and in there try to reconnect your git integration

**Reply by monisnap-julien** [reply]
Ok seems like it's indeed deactivated, it wasn't the case yesterday so it's a bit weird 🤔 
But anyway, now it's working so thank you for the answer !
